scripts/serve.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const { 
+	execute,
+	sleep,
+	watchForBuild,
+	buildTheme,
+	serveApp,
+	watchTheme,
+} = require('./build.kit');
+
+const cmdArgs = require('command-line-args');
+
+const { prod } = cmdArgs([{
+	name: 'prod',
+	type: Boolean,
+	defaultValue: false
+}]);
+
+const serveOptions = [];
+if (prod) {
+	serveOptions.push('--prod');
+}
+
+async function main() {
+	await buildAndWatch('qgrid-core');
+	await buildAndWatch('qgrid-plugins');
+	await buildAndWatch('ngx-qgrid');
+	await buildAndWatch('ngx-qgrid-plugins');
+	await buildAndWatch('ng2-qgrid');
+	await buildAndWatch('ng2-qgrid-theme-basic');
+	await buildAndWatch('ng2-qgrid-theme-material');
+	await buildAndWatch('ng2-qgrid-app');
+}
+
+async function buildAndWatch(project) {
+	await buildProject(project);
+	await watchForBuild(project);
+}
+
+async function buildProject(project) {
+	switch (project) {
+		case 'qgrid-core':
+		case 'qgrid-plugins':
+		case 'ngx-qgrid':
+		case 'ngx-qgrid-plugins':
+		case 'ng2-qgrid': {
+			execute(`cd packages/${project} && yarn build --watch`);
```

Comment:
why just now yarn build:watch where build:watch is ng build --watch or rollup --watch?